web/src/lib/components/shared-components/change-location.svelte

```diff
@@ -38,8 +62,82 @@
       dispatch('cancel');
     }
   };
+
+  const showLocation = (name: string, admin1Name?: string, admin2Name?: string): string => {
```

Comment:
Nit: technically, `getLocation` makes more sense because it returns something and doesn't actually show it